### ISSUE TITLE
can't submit ticket changes without a comment

### ISSUE THREAD

**Issue by phik** [zendesklabs/oops_app#20]
Our settings page says "Version 1.0 - Last updated 3 days ago", but I can't figure out what exactly that includes.  Does this have all of the latest fixes live in the marketplace, or do I need to install a private version of the app?

When I try to submit a change to a ticket without adding a comment (e.g. just changing priority), I get the "Oops: This app has prevented the submission of the ticket." error dialog.

If I add a comment, then I get the nice new modal dialog, and it works fine.

**Comment by peterlynch** [comment]
Yes, it is a bit confusing the version number was not changed with the latest release.
